kolibri/content/test/test_channel_import.py

```diff
@@ -295,10 +295,9 @@ def test_base_table_mapper(self, apps_mock, tree_id_mock, BridgeMock):
 
 class MaliciousDatabaseTestCase(TestCase):
 
-    @patch('kolibri.content.utils.channel_import.set_leaf_node_availability_from_local_file_availability')
```

Comment:
Almost every test in this TestCase is failing. Errors are like this:

https://travis-ci.org/learningequality/kolibri/jobs/365258329#L1375

```
    def get_content_storage_file_path(filename, datafolder=None):
>       assert VALID_STORAGE_FILENAME.match(filename), "'{}' is not a valid content storage filename".format(filename)
E       AssertionError: 'e00699f859624e0f875ac6fe1e13d648.' is not a valid content storage filename

``

Reply:
_sigh_ will fix.